packages/base/adapters/evm/wagmi/client.ts

```diff
@@ -123,11 +135,20 @@ export class Web3Modal extends Web3ModalScaffold {
 
           resolve({ approvedCaipNetworkIds: undefined, supportsAllNetworks: true })
         })
+      }
     }
 
-    const connectionControllerClient: ConnectionControllerClient = {
+    this.connectionControllerClient = {
       connectWalletConnect: async onUri => {
-        const connector = wagmiConfig.connectors.find(
+        const siweConfig = this.options?.siweConfig
+
+        if (!this.wagmiConfig) {
```

Comment:
Should we do this control in somewhere on parent scope?

Reply:
Instead of checking `if(this.wagmiConfig)` in every function, it could be done in upper scope I mean